### ISSUE TITLE
Foot colorscheme

### ISSUE THREAD

**Issue by FunctionalHacker** [catppuccin/catppuccin#24]
Hi, I'm a user of the [foot](https://codeberg.org/dnkl/foot) terminal emulator and I created the colorscheme in foot.ini format. I didn't want to create a pull request since it seems you like to use separate repositories for these kinds of integrations

```
[colors]
foreground	= D7DAE0
background	= 1E1E28
regular0	= 6E6C7C
regular1	= E28C8C
regular2	= B3E1A3
regular3	= EADDA0
regular4	= A4B9EF
regular5	= C6AAE8
regular6	= F0AFE1
regular7	= D7DAE0
bright0		= 6E6C7C
bright1		= E28C8C
bright2		= B3E1A3
bright3		= EADDA0
bright4		= A4B9EF
bright5		= C6AAE8
bright6		= F0AFE1
bright7		= D7DAE0
```

**Comment by FunctionalHacker** [comment]
It seems that the colorscheme has changed

Here's an updated foot configuration

```
[colors]
foreground 	= D9E0EE
background 	= 1E1D2F
regular0 	= 6E6C7E
regular1 	= F28FAD
regular2 	= ABE9B3
regular3 	= FAE3B0
regular4 	= 96CDFB
regular5 	= F5C2E7
regular6 	= 89DCEB
regular7 	= D9E0EE
bright0 	= 988BA2
bright1 	= F28FAD
bright2 	= ABE9B3
bright3 	= FAE3B0
bright4 	= 96CDFB
bright5 	= F5C2E7
bright6 	= 89DCEB
bright7 	= D9E0EE
```

**Comment by VictorTennekes** [comment]
This can be closed in favor of #79

**Comment by FunctionalHacker** [comment]
Thanks for the hwads up
